mappings/net/minecraft/client/particle/ParticleDrip.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bmv
+CLASS none/bmv net/minecraft/client/particle/ParticleDrip
```

Comment:
What's the difference between this and `Droplet` is it water vs lava dripping? If so, `WaterDrip` / `LavaDrip` instead maybe?